yelp-communities.php

```diff
@@ -231,9 +231,11 @@ public function yc_results( $data ) {
 					bottom: 0;
 					right: 0;
 				}
-				.yelp-branding img {
+				a.yelp-branding img {
```

Comment:
We have to add the element selector because the Twenty Seventeen theme includes a box-shadow on all images in .entry-content